<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialogstatistics</class>
 <widget class="QDialog" name="Dialogstatistics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>831</width>
    <height>676</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>831</width>
    <height>676</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>831</width>
    <height>676</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>统计信息</string>
  </property>
  <property name="windowIcon">
   <iconset resource="hotel.qrc">
    <normaloff>:/new/prefix1/Images/Common/LOGO.png</normaloff>:/new/prefix1/Images/Common/LOGO.png</iconset>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>801</width>
     <height>561</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>当月收入</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>591</width>
       <height>541</height>
      </rect>
     </property>
     <row>
      <property name="text">
       <string>当月总收入</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>1</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>2</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>3</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>4</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>5</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>6</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>7</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>8</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>9</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>10</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>11</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>12</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>13</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>14</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>15</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>16</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>17</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>18</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>19</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>20</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>21</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>22</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>24</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>25</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>26</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>27</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>28</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>30</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>31</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>房价收入</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>额外消费收入</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>总入收</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>当天收入来源</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>541</width>
       <height>541</height>
      </rect>
     </property>
     <row>
      <property name="text">
       <string>当天收入统计</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>1</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>2</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>3</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>4</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>5</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>6</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>7</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>8</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>9</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>10</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>11</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>12</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>13</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>14</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>15</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>16</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>17</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>18</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>19</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>20</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>21</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>22</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>23</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>24</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>25</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>26</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>27</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>28</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>29</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>30</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>31</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>32</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>33</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>34</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>35</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>36</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>37</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>38</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>39</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>40</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>41</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>42</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>43</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>44</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>45</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>46</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>47</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>48</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>49</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>50</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>51</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>52</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>53</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>55</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>55</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>56</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>57</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>58</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>59</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>60</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>61</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>62</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>63</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>65</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>房间号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>房价收入</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>额外消费收入</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>总收入</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>当年收入</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>481</width>
       <height>421</height>
      </rect>
     </property>
     <row>
      <property name="text">
       <string>当年统计收入</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>1月</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>2月</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>3月</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>4月</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>5月</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>6月</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>7月</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>8月</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>9月</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>10月</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>11月</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>12月</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>新建列</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>新建列</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>新建列</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>811</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/Images/Common/statistics.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="hotel.qrc"/>
 </resources>
 <connections/>
</ui>
